### PR TITLE
psc-18-timetable vc

### PR DESCRIPTION
**Pull request by MarkiyanVytrykush**

### CODE REVIEW

File: Uni/Presentation/Dashboard Story/Shedule Story/SсheduleViewController.swift [hunk 1]
```diff
@@ -12,12 +12,28 @@ import UIKit
...
     override func viewDidLoad() {
+        
         super.viewDidLoad()
-
-        presenter.onViewDidLoad()
+                let firstViewController = UIViewController()
+                let secondViewController = UIViewController()
+                let thirdViewController = UIViewController()
+                let fourthViewController =  UIViewController()
+                firstViewController.title = "Monday"
```
Comment (BohdanDatskiv) [inline Uni/Presentation/Dashboard Story/Shedule Story/SсheduleViewController.swift]: presenter should update views

File: Uni/Coordinator/AppCoordinator.swift [hunk 2]
```diff
@@ -22,6 +22,7 @@ final class AppCoordinator: AbstractCoordinator, CoordinatorProtocol {
         super.init(with: config.navigation)
         window.rootViewController = navigation
         window.makeKeyAndVisible()
+        navigation.setNavigationBarHidden(true, animated: false)
```
Comment (Pryssss) [inline Uni/Coordinator/AppCoordinator.swift]: if u hide navigation, navigation won't be in all app?

File: Uni/Presentation/Dashboard Story/Shedule Story/SсheduleViewController.swift [hunk 3]
```diff
@@ -12,12 +12,28 @@ import UIKit
...
+                secondViewController.title = "Tuesday"
+                thirdViewController.title = "Wednesday"
+                fourthViewController.title = "Thursday"
+                let ps = PresenterViewController.init(viewControllers:  [firstViewController, secondViewController, thirdViewController, fourthViewController])
+                addChild(ps)
```
Comment (Pryssss) [inline Uni/Presentation/Dashboard Story/Shedule Story/SсheduleViewController.swift]: maybe we rather make function for creating all these and proper mvp if needed

File: Uni/Presentation/Dashboard Story/TimeTableVC/PresenterViewController.swift [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+import Parchment
+import UIKit
+import Foundation
+
+final class PresenterViewController: FixedPagingViewController {
+
+    override init(viewControllers: [UIViewController]) {
+        super.init(viewControllers: viewControllers)
+        self.selectedFont = .systemFont(ofSize: 15, weight: .medium)
```
Comment (Pryssss) [inline Uni/Presentation/Dashboard Story/TimeTableVC/PresenterViewController.swift]: plz move colors to assets

File: Uni/Presentation/Dashboard Story/Shedule Story/SсheduleViewController.swift (hunk 1 above)
Comment (MarkiyanVytrykush) [reply Uni/Presentation/Dashboard Story/Shedule Story/SсheduleViewController.swift]: fixed

File: Uni/Coordinator/AppCoordinator.swift (hunk 2 above)
Comment (MarkiyanVytrykush) [reply Uni/Coordinator/AppCoordinator.swift]: fixed

File: Uni/Presentation/Dashboard Story/Shedule Story/SсheduleViewController.swift (hunk 3 above)
Comment (MarkiyanVytrykush) [reply Uni/Presentation/Dashboard Story/Shedule Story/SсheduleViewController.swift]: fixed

File: Uni/Presentation/Dashboard Story/TimeTableVC/PresenterViewController.swift (hunk 4 above)
Comment (MarkiyanVytrykush) [reply Uni/Presentation/Dashboard Story/TimeTableVC/PresenterViewController.swift]: fixed

**Pull request closed without merging**